### DISCUSSION TITLE
Is it possible to use an annotated multiome dataset to annotate scATAC cell types?

### DISCUSSION THREAD

**Discussion by sabrinacamp2** [stuart-lab/signac#1194] [Q&A] [answered]
Hi,

I am working with a dataset that is comprised of **scATAC samples only**, no scRNA. Cell type annotation in scATAC data beyond broad cell types has proved challenging. Some genes can be highly accessible in multiple cell types, but only be expressed in one of them due to differential activation by TFs among other possible biological explanations. 

One example of this was found in an [Immunity paper in 2016](https://www.cell.com/immunity/pdfExtended/S1074-7613(16)30439-3). They found activation induced genes such as _Prf1_ to be similarly accessible in naive, effector, and memory CD8+ T cells despite very different expression. This makes cell type annotation particularly challenging, as an inferred _Prf1_ gene activity in these three cell types/states would lead someone to believe the expression is similar in each cell type.

For reasons mentioned above, using the correlation between gene activity and the expression from a reference scRNA dataset to annotate cell types might present problems for attempting a more detailed cell type annotation. 

I wondered if there are any thoughts on how someone could use a **multiome dataset** with cell type annotations (from the RNA, since the markers are more specific and well defined) to annotate a scATAC only dataset. More specifically, could this annotation take advantage of the chromatin accessibility information in the multiome dataset to annotate the scATAC data, bypassing the potential issues with gene activity scores?

Appreciate any thoughts,
Sabrina

**Comment by timoast** [comment, marked as answer]
Yes you can, example shown here: https://satijalab.org/signac/articles/integrate_atac.html#reference-mapping

**Reply by sabrinacamp2** [reply]
Thank you!
